techtv2ovs/utils.py

```diff
@@ -198,8 +248,14 @@ def process_files(self, ttv_video, s3videos):
             ttv_video (TechTVVideo): The TechTVVideo object to process
             s3videos (list): List of video file S3 objects
         """
-        self.process_thumbs(ttv_video)
-        process_videofiles.delay(ttv_video.id, s3videos, self.aws)
+        try:
+            self.process_thumbs(ttv_video)
+            self.process_captions(ttv_video)
+            process_videofiles.delay(ttv_video.id, s3videos, self.aws)
+        except Exception as exc:
+            ttv_video.status = ImportStatus.ERROR
+            ttv_video.errors += '{}\n\n'.format(str(exc))
```

Comment:
It might be useful to include the stack trace here too, for example: https://stackoverflow.com/questions/4564559/get-exception-description-and-stack-trace-which-caused-an-exception-all-as-a-st#answer-35712784 https://docs.python.org/3/library/traceback.html#traceback.format_exception

Reply:
✅